crates/chia-puzzles/src/puzzles/nft.rs

```diff
@@ -57,6 +135,36 @@ pub struct NftRoyaltyTransferPuzzleArgs {
     pub trade_price_percentage: u16,
 }
 
+impl NftRoyaltyTransferPuzzleArgs {
+    pub fn new(
+        launcher_id: Bytes32,
+        royalty_puzzle_hash: Bytes32,
```

Comment:
is this not a `TreeHash`?

Reply:
It has to be encoded as the puzzle hash atom rather than the puzzle itself